README.md

```diff
@@ -0,0 +1,28 @@
+![](./media/Standard_4_Person_Clover_Animation.gif)
```

Comment:
If you think this animated gif is too distracting, we can definitely remove it.   So no worries if you prefer to delete it.